package.json

```diff
@@ -11,7 +11,7 @@
     "build": "NODE_ENV=production webpack -p --config webpack.production.config.js",
     "test": "babel-node testrunner",
     "examples": "webpack-dev-server --config examples/webpack.config.js --content-base examples",
-    "prepare": "babel ./src/ -d ./lib/"
+    "prepublish": "babel ./src/ -d ./lib/"
```

Comment:
Reverting this back to what it used to be in the base repo since we're not gonna use it anymore